docs/src/js/components/ElementsComment.react.js

```diff
@@ -96,7 +195,72 @@ export default class ElmentsComment extends React.Component {
                 <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
                     {commentSample}
                 </Highlighter>
+
+                {/* Editable Comment */}
+                <Header anchor="divider" size="large" style={{ marginTop: '55px' }} sub={true}>
+                    Editable Comments
+                    <Header.Subheader>
+                        A comment can be editable and deletable.
+                    </Header.Subheader>
+                </Header>
+
+                <Comment
+                    canEdit
+                    canRemove
+                    isEditable
+                    name="Joe Smith"
+                    onRemove={this._onRemoveComment}
+                    onSaveEdit={this._onSaveComment}
+                    text={editableCommentText}
+                    time={1536941364}
+                >
+                    {editableCommentText}
+                </Comment>
+
+                <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
+                    {editableCommentSample}
+                </Highlighter>
+
+                <Banner
+                    id="remove-success"
+                    level="success"
+                    isOpen={isRemoveBannerOpen}
+                    message="Comment Removed!"
+                    onClose={this._onCloseRemoveBanner}
+                    title="Comment Removed"
+                    type="notification"
+                />
+
+                <Banner
```

Comment:
How are the Banners going to be rendered, location-wise, in the UI?

Reply:
On the UI docs page?  Should be up near the top of the view port as normal.  I can demo this live if you'd like.  Just to reiterate, I only wired up banners for demo purposes on the documentation page.  That is 100% external to the component itself ... it's 100% up to the consumer (as it should be in my opinion).